### ISSUE TITLE
ZUGFeRD-2.1.1 - BR-DE-15 Das Element „Buyer reference“ (BT-10) muss übermittelt werden.

### ISSUE THREAD

**Issue by BoHamburg** [ZUGFeRD/mustangproject#236]
I get the error message when the element BT-10 (Buyer reference) is not specified.

In the case of an XRechnung, this makes sense, since the buyer reference is the Leitweg-ID ... in the current ZUGFErD documentation, the specification is given as 0 ... 1, i.e. not absolutely necessary. 

The command line output under Windows 10 Prof. does not seem to be UTF-8, see redirected output as an attachment.

[output.txt](https://github.com/ZUGFeRD/mustangproject/files/6244758/output.txt)

Thank you for the Mustang project :-)

**Comment by heisej** [comment]
You are correct that the element is optional for the normal ZUGFeRD profiles (Basic, EN16931, Comfort,...).

 But from the output you attached it seems you are using the XRechnung reference profile (urn:cen.eu:en16931:2017#compliant#urn:xoev-de:kosit:standard:xrechnung_2.0) from ZUGFeRD and for that the element is mandatory as the profile conforms to the CII format of the XRechnung specification.

For ZUGFeRD files according the normal profiles the ZUV still returns a warning note if the element is missing, but no hard error.

**Comment by BoHamburg** [comment]
I went to the profile because I got another error, "[BR-DE-21] The element" Specification identifier "(BT-24) should correspond syntactically to the identifier of the XRechnung standard." 

So It's posible the validator has criticized some XRechnung points for a ZUGFErD invoice ... 
[output.txt](https://github.com/ZUGFeRD/mustangproject/files/6245168/output.txt)

**Comment by heisej** [comment]
If you look at the output closely, you will notice that this it not reported as error, but just a notice which will not report the document as invalid. 

This had been discussed before in issue #172.

**Comment by BoHamburg** [comment]
I have always looked at the "failed" ... not below in the message text with "error" and "notice"... ARGH!

**Comment by heisej** [comment]
Hehe. I had that myself the first time I tried running the validator on one of my generated files, but noticed that the summary status at the very end of the output showed "valid". Then I had a closer look at the output before that and saw that it's only a notice not a hard error.

**Comment by jstaerk** [comment]
then lets keep the issue open for a whileif someone comes up with an idea how to improve it.
